rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -12,18 +11,13 @@
 __version__ = "0.23.0-alpha.1+dev"
 __version_info__ = (0, 23, 0, "alpha.1")
 
-if sys.version_info < (3, 9):
-    warnings.warn(
-        "Python 3.8 is past EOL (https://devguide.python.org/versions/). Rerun version 0.21 will drop support/testing of Python 3.8.",
-        DeprecationWarning,
-    )
```

Comment:
Wat happens if someone tries to run Rerun with python 3.8 now?  Should we still print a warning?

Reply:
Good catch! I replaced that by raising an exception, but ruff deleted the entire thing because, well, the `if` is never supposed to be true. And it's right in a sense: no tooling will ever allow installing the `rerun-sdk` package in a <3.9 environment. I'll put it back regardless with a ruff exception, just to be safe.